### PR TITLE
Fix style on imports

### PR DESCRIPTION
**Pull request by toslunar**

### CODE REVIEW

File: chainer/functions/array/as_strided.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
+import numpy as np
+import six.moves
```
Comment (niboshi) [inline chainer/functions/array/as_strided.py]: How about `import six`?

File: chainer/datasets/image_dataset.py [hunk 2]
```diff
@@ -1,17 +1,18 @@
 import os
 
+import bisect
```
Comment (niboshi) [inline chainer/datasets/image_dataset.py]: Can be merged with `import os` above?

**Comment by toslunar** [comment]
I fixed codes as suggested.

**Comment by niboshi** [comment]
Thanks, LGTM
Jenkins, test this please

**Comment by niboshi** [comment]
LGTM

**Pull request merged**